pyproject.toml

```diff
@@ -87,6 +87,7 @@ PyYAML = "~5.4.1"
 social-auth-app-django = "^4.0.0"
 # Rendering of SVG images (for rack elevations, etc.)
 svgwrite = "~1.4.1"
+django-redis = "^4.12.1"
```

Comment:
Can you please alpha sort this in this section of the file and add a comment above it? (I suggested a comment)

```suggestion
# Redis cache for Django used for distributed locking
django-redis = "^4.12.1"
```